src/components/ec-inline-actions/ec-inline-actions.vue

```diff
@@ -88,6 +88,9 @@ export default {
     },
   },
   methods: {
+    allowedActions(item) {
```

Comment:
`allowedActions` is a misleading name. It sounds like a getter which gives you all allowed actions. 

What about `callAction(item)` or doAction(item)

Reply:
Amended to "doAction". Thank you